Promact.Oauth.Server/src/Promact.Oauth.Server/wwwroot/app/users/user-add/user-add.component.ts

```diff
@@ -71,31 +72,34 @@ export class UserAddComponent {
     }
 
     checkEmail(email) {
-        this.isEmailExist = false;
-
-        this.userService.findUserByEmail(email + "@promactinfo.com").subscribe((isEmailExist) => {
-            if (isEmailExist) {
-                this.isEmailExist = true;
-            }
-            else {
-                this.isEmailExist = false;
+        if (email !== "" && email !== undefined) {
+            this.isEmailExist = false;
+            this.userService.findUserByEmail(email + "@promactinfo.com").subscribe((isEmailExist) => {
```

Comment:
Why @promactinfo.com is hard-coded at this place? It should be checked on the server side.